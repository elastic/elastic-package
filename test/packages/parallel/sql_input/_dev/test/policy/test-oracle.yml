vars:
  driver: "oracle"
  hosts:
    - root:test@tcp(localhost)/
  # FIXME: This doesn't work as a multiline block.
  sql_query: SELECT file_name, file_id, tablespace_name, bytes, status, maxbytes, user_bytes, online_status FROM sys.dba_data_files
