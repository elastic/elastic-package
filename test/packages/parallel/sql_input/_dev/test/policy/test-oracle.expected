inputs:
    - data_stream:
        namespace: ep
      meta:
        package:
            name: sql_input
      name: test-oracle-sql_input
      streams:
        - data_stream:
            dataset: sql_input.sql_query
          driver: oracle
          hosts:
            - root:test@tcp(localhost)/
          metricsets:
            - query
          period: 10s
          sql_query: SELECT file_name, file_id, tablespace_name, bytes, status, maxbytes, user_bytes, online_status FROM sys.dba_data_files
          sql_response_format: variables
      type: sql/metrics
      use_output: default
output_permissions:
    default:
        _elastic_agent_checks:
            cluster:
                - monitor
        _elastic_agent_monitoring:
            indices: []
        uuid-for-permissions-on-related-indices:
            indices:
                - names:
                    - metrics-*-*
                  privileges:
                    - auto_configure
                    - create_doc
secret_references: []
